lektor_tailwind.py

```diff
@@ -22,51 +22,46 @@ def __init__(self, env, id):
         self.css_path = config.get("css_path", "static/style.css")
         self.input_css = os.path.join(self.env.root_path, "assets", self.css_path)
         self.tailwind: subprocess.Popen | None = None
+        self.config_file = "tailwind.config.js"
 
     def on_setup_env(self, **extra):
         self.init_tailwindcss()
 
     def init_tailwindcss(self):
         if not os.path.exists(self.tailwind_bin):
             install(bin_path=self.tailwind_bin)
-        filename = "tailwind.config.js"
-        if not os.path.exists(os.path.join(self.env.root_path, filename)):
+        if not os.path.exists(os.path.join(self.env.root_path, self.config_file)):
             subprocess.run(
                 [self.tailwind_bin, "init"], check=True, cwd=self.env.root_path
             )
 
     def _run_watcher(self, output_path: str):
         if not self.input_exists():
             return
+        self.tailwind = subprocess.Popen(
+            self._get_tailwind_args(output_path, "-w",
+                                    *(["--minify"] if os.environ.get("NODE_ENV") == "production" else [])),
+            cwd=output_path,
+        )
 
-        cmd = [
-            self.tailwind_bin,
-            "--input",
-            self.input_css,
-            "--output",
-            os.path.join(output_path, self.css_path),
-            "--watch",
-        ]
-        if os.environ.get("NODE_ENV") == "production":
-            cmd.append("--minify")
-
-        self.tailwind = subprocess.Popen(cmd, cwd=self.env.root_path)
+    def _get_tailwind_args(self, output_path, *extra_args):
+        return [self.tailwind_bin,
+                "-c",
+                os.path.join(self.env.root_path, self.config_file),
+                "-i",
+                self.input_css,
+                "-o",
+                os.path.join(output_path, self.css_path),
+                *extra_args,]
 
     def input_exists(self) -> bool:
         return os.path.exists(self.input_css)
 
     def compile_css(self, output_path: str):
         subprocess.run(
-            [
-                self.tailwind_bin,
-                "-i",
-                self.input_css,
-                "-o",
-                os.path.join(output_path, self.css_path),
-                "--minify",
-            ],
+            self._get_tailwind_args(output_path, "--minify"),
             check=True,
-            cwd=self.env.root_path,
+            cwd=output_path,
```

Comment:
But this method is run `before_build`, I doubt if it works

Reply:
You're right, fixed.